fix various delegate docs (#599)
harness/developer-hub: docs/continuous-delivery/cd-advanced/cd-kubernetes-category/add-and-override-values-yaml-files.md

   
   ![](./static/add-and-override-values-yaml-files-33.png)

> 5. In **Specify Values YAML Store**, select the Git repo provider you're using and then create or select a Connector to that repo. The different Connectors are covered in [Connect to a Git Repo](../../../platform/7_Connectors/connect-to-code-repo.md).

Reviewer: ```suggestion
5. In **Specify Values YAML Store**, select your Git repository provider and then create or select a connector to that repository. The different connectors are covered in [Connect to a Git Repo](../../../platform/7_Connectors/connect-to-code-repo.md).
```

> 6. If you haven't set up a Harness Delegate, you can add one as part of the Connector setup.

Reviewer: ```suggestion
6. If you haven't yet set up a delegate, you can add one when you set up a connector.
```

>     This process is described in [Kubernetes deployment tutorial](../../onboard-cd/cd-quickstarts/kubernetes-cd-quickstart.md), [Helm Chart  deployment tutorial](../../onboard-cd/cd-quickstarts/helm-cd-quickstart.md) and [Install a Kubernetes Delegate](/docs/platform/2_Delegates/advanced-installation/install-a-kubernetes-delegate.md).

Reviewer: ```suggestion
    This process is described in [Kubernetes deployment tutorial](../../onboard-cd/cd-quickstarts/kubernetes-cd-quickstart.md), [Helm Chart  deployment tutorial](../../onboard-cd/cd-quickstarts/helm-cd-quickstart.md), and [Install a Kubernetes Delegate](/docs/platform/2_Delegates/advanced-installation/install-a-kubernetes-delegate.md).
```

> 7. Once you've selected a Connector, click **Continue**.

Reviewer: ```suggestion
7. After you select a Connector, click **Continue**.
```

> 8. In **Manifest Details**, you tell Harness where the values.yaml is located.

Reviewer: ```suggestion
8. In **Manifest Details**, specify the location of the values.yaml file.
```

> 9. In **Manifest Identifier**, enter a name that identifies the file, like **values**.

Reviewer: ```suggestion
9. In **Manifest Identifier**, enter a descriptive label for the file, like **values**.
```

> 10. In **Git Fetch Type**, select a branch or commit Id for the manifest, and then enter the Id or branch.

Reviewer: ```suggestion
10. In **Git Fetch Type**, select a branch or commit ID for the manifest. Then enter the ID or branch.
```

11.  For **Specific Commit ID**, you can also use a [Git commit tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging).

> 12. In **File Path**, enter the path to the values.yaml file in the repo.

Reviewer: ```suggestion
12. In **File Path**, enter the path to the values.yaml file in the repository.
```

    
    You can enter multiple values file paths by clicking **Add File**. At runtime, Harness will compile the files into one values file.
    
    If you use multiple files, the highest priority is given from the last file, and the lowest priority to the first file. For example, if you have 3 files and the second and third files contain the same `key:value` as the first file, the third file's `key:value` overrides the second and first files.
    
    ![](./static/add-and-override-values-yaml-files-34.png)

> 13. Click **Submit**. The values file(s) are added to the Service.

Reviewer: ```suggestion
13. Click **Submit**. The values files you specified are added to the service.
```
